frontend/packages/dev-console/src/components/edit-application/EditApplicationForm.tsx

```diff
@@ -43,36 +44,43 @@ const EditApplicationForm: React.FC<FormikProps<FormikValues> & EditApplicationF
       <PageHeading title={createFlowType} />
       <FlexForm onSubmit={handleSubmit}>
         <FormBody flexLayout>
-          {createFlowType !== CreateApplicationFlow.Container &&
-            createFlowType !== CreateApplicationFlow.JarUpload && (
+          {createFlowType !== i18n.t(EditFlowPageTitle[CreateApplicationFlow.Container]) &&
+            createFlowType !== i18n.t(EditFlowPageTitle[CreateApplicationFlow.JarUpload]) && (
```

Comment:
Again, this is a component, favour the hook `t`

Reply:
+1. `useTranslation` is already used here.

But IMHO we should also not check this enum type against translated values. Also if the enum values are the actual english strings we should consider them as "keys". I would recommend to reverse this changes here as well.

If also worth to change the Props definition from `createFlowType: string` to `createFlowType: CreateApplicationFlow` to make this more clear.